### ISSUE TITLE
ISSUE: RELOAD dialog appears when MONITORing a file

### ISSUE THREAD

**Issue by Daklu** [notepad-plus-plus/notepad-plus-plus#5586]
### Description of the Issue
When monitoring a file, the reload dialog appears when the file is updated.  This behavior is not observed in v7.6.3, and the behavior is slightly different in v7.6.4 than it is in 7.6.6.

### Steps to Reproduce the Issue
1. Monitor a file
2. Change the contents of the file externally

### Expected Behavior
The file is updated with no reload dialog

### Actual Behavior
v7.6.3 - No reload dialog appears.  (Expected behavior.)
v7.6.4 - Sometimes the reload dialog appears, sometimes it does not.  Can't cleanly repro.  May be related to whether or not Notepad++ has focus.
v7.6.6 - The reload dialog appears every time the window is updated.

### Debug Information
Notepad++ v7.6.6   (64-bit)
Build time : Apr  3 2019 - 23:52:32
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll

**Comment by MetaChuh** [comment]
@Daklu 

thank you for your debug information and details.

we can currently not reproduce this on 7.6.6. x64 or x86.
(see screencast, with working monitoring on 7.6.6 x64)
![Imgur](https://i.imgur.com/dfLdYUp.gif)

but we had a [second report about this at the notepad++ community](https://notepad-plus-plus.org/community/topic/17513/monitor-tail-f-prompts-to-reload-file), so we suppose it might be a setting, or leftover files from an old installation that could cause this.

**please test the following, to verify, if a clean notepad++ (without any plugins or settings that could cause this) would monitor as expected on your system:**

* download the notepad++ 7.6.6. 64 bit portable version from [>>> here <<<](https://notepad-plus-plus.org/repository/7.x/7.6.6/npp.7.6.6.bin.x64.zip).
(the portable version does not require any installation and runs completely independent and isolated from your installed version. all custom settings, as well as any stuff you might want to try at the portable version, will not modify your installed version)

* extract `npp.7.6.6.bin.x64.zip` to e.g. your desktop.

* **important note:** make sure to close all instances of notepad++ that might be running, before starting the portable version at the next step, to be sure you are using the portable version for this test.

* open the extracted `npp.7.6.6.bin.x64` folder and start this portable version by double-clicking on `notepad++.exe` inside this folder.
(note: notepad++.exe will just be seen as notepad++, if you have enabled to hide all known file extensions at your explorer settings)

* retry monitoring (tail -f) on the same files using this portable version.
if it works, we will have to find out the settings or file differences between the clean portable and the installed version.

many thanks and best regards

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by SinghRajenM** [comment]
Duplicate of #4847

Somewhat related: #4325

PR #4527 might have fixed this, but not sure.

This seems to be a thread timing issue.... Easy reproduction steps are -

1. Open document in n++ and click on monitoring icon.
2. Now open the same document in windows built-in notepad and 
3. Now keep pressing <kbd>ctrl</kbd>+<kbd>s</kbd> or (just keep <kbd>ctrl</kbd>+<kbd>s</kbd> pressed, don't release any key.)
4. Voila, you got what you wanted.

May be I will look into this once I find some time.

**Comment by MetaChuh** [comment]
i can reproduce this now on both 7.6.6 and 7.6.3 by monitoring `ping -t 8.8.8.8 > ping.log`.
(see screencast below)

open cmd and run `ping -t 8.8.8.8 > ping.log`
monitor ping.log in notepad++ and it will **not update without focus change** until ping.log is closed.
changing focus results in a reload prompt.

note:
only seems happen on windows 10.
windows 7 does not prompt for reload, but monitoring a file also only gets updated when changing focus or the file is closed.

![7 6 6  monitoring bug](https://user-images.githubusercontent.com/45773305/56852219-a501c900-6918-11e9-8af8-bd976c52ba77.gif)

**Comment by SinghRajenM** [comment]
What about using this [nightly build](https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/24142247/job/apf733egsis6b5ih/artifacts)? I must have fixed it. FYR, I can't reproduce with above nightly build.

**Comment by MetaChuh** [comment]
@SinghRajenM 
my apologies for the current delay.
please see https://github.com/notepad-plus-plus/notepad-plus-plus/pull/5591
